src/pages/CertificateLanding/CertificateLandingPage.tsx

```diff
@@ -198,7 +211,7 @@ const CertificateLandingPage: React.FC = () => {
                             </div>
                         </AccordionComponent>
                     </div>
-                    {certificate?.status !== 'used' && user?.complete_onboarding && (
+                    {certificate?.status !== 'used' && user?.complete_onboarding && !isCertificatedDisabled() && (
```

Comment:
certificate?.status !== 'used' is useless as it chacks in isCertificatedDisabled